spec/lib/video_info/providers/vkontakte_spec.rb

```diff
@@ -142,4 +143,32 @@
     its(:title)            { should eq 'SpaceGlasses are the future of computing' }
   end
 
+  context "with valid video and connection timeout" do
+    subject { VideoInfo.new('http://vk.com/video-54799401_165822734') }
+
+    describe '#title' do
+      before do
+        stub_request(:any, /.*vk.com.*/).to_timeout
+      end
+
+      it 'raises VideoInfo::HttpError exception' do
+        expect { subject.title }.to raise_error VideoInfo::HttpError
+      end
+    end
+  end
+
+  context "with valid video and OpenURI::HTTPError exception" do
+    subject { VideoInfo.new('http://vk.com/video-54799401_165822734') }
+
+    describe '#title' do
+      before do
+        stub_request(:any, /.*vk.com.*/).to_raise(OpenURI::HTTPError.new("error", :nop))
```

Comment:
Line is too long. [88/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.